crates/ruff_linter/src/checkers/ast/annotation.rs

```diff
@@ -85,6 +88,9 @@ impl AnnotationContext {
         if flake8_type_checking::helpers::runtime_required_function(
             function_def,
             &settings.flake8_type_checking.runtime_required_decorators,
+            &settings
+                .flake8_type_checking
+                .runtime_required_decorator_classes,
```

Comment:
Nit: I suggest passing the `flake8_type_checking` settings instead of passing every setting individually

Reply:
I wanted to make as few changes as possible, but I can change this, of course.

Reply:
I have changed this, and the callers, too.